src/python/bcc/__init__.py

```diff
@@ -950,7 +950,16 @@ def trace_fields(self, nonblocking=False):
                 ts_end = line.find(":")
                 pid, cpu, flags, ts = line[:ts_end].split()
                 cpu = cpu[1:-1]
-                msg = line[ts_end + 4:]
+                # line[ts_end:] will have ": [sym_or_addr]: msgs"
+                # For trace_pipe debug output, the addr typically
+                # is invalid (e.g., 0x1). For kernel 2.12 or earlier,
```

Comment:
comment doesn't look right. kernel 2.12 ?

Reply:
Right. Sometimes long commit msg/comment for a tiny change does make it easy to have bugs in the comments rather than code :-). Just fixed and pushed a new commit.